pkgs/top-level/all-packages.nix

```
@@ -20780,6 +20780,10 @@ with pkgs;
 
   ghcid = haskellPackages.ghcid.bin;
 
+  gr-framework = libsForQt5.callPackage ../development/libraries/gr-framework {
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
```

```suggestion
    inherit (darwin.apple_sdk_11_0) stdenv;
```
that is already done behind the scenes

---

This doesn't work as `rewrite-tbd` is Darwin specific:

https://github.com/NixOS/nixpkgs/blob/13c9bc70606f9544ed21501cf1d5384c1731b77c/pkgs/os-specific/darwin/rewrite-tbd/default.nix#L28

---

This works because darwin.apple_sdk_11_0.stdenv falls back to stdenv on linux